site-content/source/modules/ROOT/pages/ecosystem.adoc

```diff
@@ -69,7 +69,9 @@ https://ignite.apache.org/[Apache Ignite,window=blank]: Use Ignite as a traditio
 
 https://github.com/Azure-Samples/cassandra-proxy[Azure-Samples/Cassandra Proxy,window=blank]: This proxy handles client connections and forwards them to two Cassandra clusters simultaneously.
 
-http://cassandra.link/[Cassandra.link,window=blank]: Curated site with tools, along with cassandra.tools.
+http://cassandra.link/[Cassandra.Link,window=blank]: Curated site with Cassandra articles, videos, and news feed.
```

Comment:
with the addition of `cassandra.tools` it means that `cassandra.link` doesn't really belong in the "Cassandra Tools" section anymore. In fact I'm not sure where we would put `cassandra.link` on this page anymore?